public/scripts/app.js

```diff
@@ -1,7 +1,105 @@
-console.log("Sanity Check: JS is working!");
+$(document).ready(function() {
+  console.log('app.js loaded!');
 
-$(document).ready(function(){
+  // make a GET request for all ski runs
+  $.ajax({
+    method: 'GET',
+    url: '/api/skiruns',
+    success: onSuccess,
+    error: function handleError (data) {
+      console.log("oops error " + data);
+    }
+  });
 
-// your code
+function onSuccess (skiruns) {
+  skiruns.forEach(function(skirun) {
```

Comment:
Nice job moving past your fear of `forEach`! (I suppose it helps that this was borrowed from Tunely...)